### ISSUE TITLE
Some question of the metric method---FID 

### ISSUE THREAD

**Issue by cookie-ke** [IIGROUP/TediGAN#18]
Thank you for your work! I try to evaluate the FID and I used the code (**fid.py**)in the link you provided. But there always gets an error. I try to run the code on colb and my own computer and the errors are shown below. I don't know why this is happening. Can I use the PyTorch version of FID instead of the TensorFlow version of FID?

![image](https://user-images.githubusercontent.com/56613566/124050401-15528a80-da4d-11eb-980c-c23d1aadcdd4.png)
![image](https://user-images.githubusercontent.com/56613566/124050832-e557b700-da4d-11eb-805d-84425613c0c5.png)

```
2021-06-30 13:57:01.685915: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using _XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
tcmalloc: large alloc 4718592000 bytes == 0x5586091f8000 @  0x7f26b949d1e7 0x7f26b6fdd46e 0x7f26b702dc7b 0x7f26b7030e83 0x7f26b703107b 0x7f26b70d2761 0x5584d699ccc0 0x5584d699ca50 0x5584d6a10be0 0x5584d6a0b4ae 0x5584d699e3ea 0x5584d6a0d32a 0x5584d6a0b4ae 0x5584d699e3ea 0x5584d6a0d32a 0x5584d6a0b4ae 0x5584d6a0b1b3 0x5584d6ad5182 0x5584d6ad54fd 0x5584d6ad53a6 0x5584d6aac723 0x5584d6aac3cc 0x7f26b8287bf7 0x5584d6aac2aa
tcmalloc: large alloc 1244758016 bytes == 0x55851527c000 @  0x7f26b949d1e7 0x7f26a01bbc05 0x7f26a0c908fe 0x7f26a306ca20 0x7f26a30d4b90 0x7f26a33328e8 0x7f26a3358926 0x7f26a335909a 0x7f26a335a00a 0x7f269ccc2136 0x7f269ccb25c5 0x7f269cd5c3ae 0x7f269cd59228 0x7f26b7d7fa50 0x7f26b92526db 0x7f26b838771f_
^C
```
![image](https://user-images.githubusercontent.com/56613566/124051025-3d8eb900-da4e-11eb-8ed5-b6badd5cb12d.png)

**Comment by weihaox** [comment]
Yes, there are many pytorch implements of FID you can use.

**Comment by cookie-ke** [comment]
Thanks a lot for your reply！
